openedx/core/djangoapps/content_tagging/migrations/0001_squashed.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 3.2.21 on 2023-10-09 23:12
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    replaces = [
+        ('content_tagging', '0001_initial'),
+        ('content_tagging', '0002_system_defined_taxonomies'),
+        ('content_tagging', '0003_system_defined_fixture'),
+        ('content_tagging', '0004_system_defined_org'),
+        ('content_tagging', '0005_auto_20230830_1517'),
+        ('content_tagging', '0006_simplify_models'),
```

Comment:
We were starting to get problems with some of these migrations, referring to models that have been deleted or depending on migrations which depended on fixtures, which were later changed or deleted (note to self: migrations should never reference fixtures). So I've squashed the migrations here which makes it much simpler and more robust. I'd really like to get the squashed version into Quince. If possible, I'd like to squash the migrations in oel_tagging too but that would be a separate PR.